src/components/Dropdown/Dropdown.tsx

```diff
@@ -21,7 +21,13 @@ export interface DropdownProps {
   initialValue?: string,
   parentQuery?: string,
   onSelect?: (value: string, index: number, focusedItemData: Record<string, unknown> | undefined) => void,
-  onToggle?: (isActive: boolean, value: string) => void,
+  onToggle?: (
+    prevValue: string,
```

Comment:
I'm wondering if some of these things need to be optional (or `undefined`). Supposing I toggle the drop-down open for the first time. There shouldn't be a previous value, a value, or any focusedItemData.

Reply:
All of the things except focusedItemData start with a default value- the strings start as '' and the index starts as -1